src/test/java/org/mockito/CustomMockMakerTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.internal.creation.bytebuddy.ByteBuddyMockMaker;
+import org.mockito.internal.creation.bytebuddy.InlineByteBuddyMockMaker;
+
+import java.util.Arrays;
+
+public final class CustomMockMakerTest {
```

Comment:
Can we also add support for `@Mock` and add a corresponding test as well?

Additionally, we are missing a bit of code coverage. Would it be possible to write some additional regression tests for these?

Reply:
I added an option for `@Mock` and a corresponding test. I did not add an option for `@Spy`, through.